manifests/z/Zheguisoft/zg-ipchat/1.00.04/Zheguisoft.zg-ipchat.locale.en-US.yaml

```diff
@@ -13,7 +13,11 @@ PackageUrl: https://www.zheguisoft.com
 License: Proprietary freeware, based on open source components
 Copyright: Shanghai Zhegui Software Co., Ltd.
 ShortDescription: zg-ipchat is a local area network chat tool, using Java Swing to write IpChat
-Description: Supports IPv6/IPv4 networks. It can realize simple text information transmission, point-to-point direct communication without intermediate server. Use "Friends List" to distinguish friends from strangers. The software interface supports Chinese and English. Operating environment,Java JRE 1.8 or higher, Windows/Linux operating system.For customized development, please contact support@zheguisoft.com
+Description: |-
+  Supports Pv6/IPv4 networks. It can realize simple text information transmission, point-to-point direct communication without intermediate server. 

```

Comment:
```suggestion
  Supports IPv6/IPv4 networks. It can realize simple text information transmission, point-to-point direct communication without intermediate server. 
```